frontend_tests/node_tests/compose.js

```diff
@@ -499,6 +499,47 @@ people.add(bob);
     assert($("#new_message_content").is_focused());
 }());
 
+(function test_finish() {
+    $("#compose_invite_users").show();
+    $("#compose-send-button").removeAttr('disabled');
+    $("#compose-send-button").focus();
+    $("#sending-indicator").hide();
+    $("#new_message_content").select(noop);
+    $("#new_message_content").val('');
+    assert(!compose.finish());
```

Comment:
I would make it more explicit that this is what is actually being tested:

~~~ js
// Test compose.finish() returns false when there is no content.
var res = compose.finish();
assert.equal(res, false);
~~~

I tend to do `assert.equal(foo, false)` when we can expect `false` to be the return value, as opposed to something that is merely false-y.  Our JS code has quirks where functions can return either `false` or `undefined`, and it's good to know when `false` is explicitly returned.